storage/innobase/xtrabackup/doc/source/backup_scenarios/incremental_backup.rst

```diff
@@ -5,7 +5,9 @@ Incremental Backup
 ==================
 
 |xtrabackup| supports incremental backups, which means that they can copy only
-the data that has changed since the last backup.
+the data that has changed since the last backup. 
+
+.. note:: You cannot take incremental backups on the MyRocks storage engine.
```

Comment:
Can we mention something like 
In each incremental backup, PXB copies the MyRocks files again irrespective if they copied in last full/incremental backup.

Reply:
Thank you. Rewritten.